getExamples.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+wget -P workspace https://github.com/INTO-CPS-Association/DTaaS-examples/archive/refs/heads/main.zip
```

Comment:
these commands work only if they are executed from root "/" directory.
It is better to make it explicit.

```sh
wget -P "/workspace" .....
unzip /workspace/.......
```sh